install_sct

```diff
@@ -538,22 +538,13 @@ pip install numpy
 if [[ -f "requirements-freeze.txt" ]]; then
   print info "Using requirements-freeze.txt (release installation)"
   pip install -r requirements-freeze.txt
-else
-  # Not a package
-  print info "Using requirements.txt (git installation)"
-  pip install -r requirements.txt
 fi
+
+pip install -r install.txt
```

Comment:
Now that `pip install -r install.txt` is outside the if/else block, it'll get run in addition to `pip install -r requirements-freeze.txt`, which I think could undo the installations of the pinned versions.

Before, when we had two steps (`pip install -r requirements.txt` and `pip install -e $SCT_DIR`), this if/else block provided two separate installation paths, which helped prevent that.